### DISCUSSION TITLE
Error after 'Append to Existing note'

### DISCUSSION THREAD

**Discussion by viking2917** [coddingtonbear/obsidian-web#82] [Q&A] [answered]
Thanks very much for this plugin!

Issue: I can create new notes, but cannot "append to existing note", in particular when the appended new note is from the same page as an existing note. 

Steps to reproduce:

1. (I have obsidian installed with a vault created).
2. install the chrome plugin, set the API key. (I have tried this on both Chrome and Brave, they both do the same thing)
3. Visit a web page, highlight some text, then do "Create new note"
4. The note is successfully created in Obsidian.
5. Highlight a different piece of text on the same page, then hit "Append to existing note"

My expectation is that my Obsidian note will then contain both pieces of highlighted text.

What actually happens is the following error:

```Could not send content to Obsidian: (Error Code 40400) Not Found```

I have not altered any of the API settings or routes, simply added my API Key to the web settings page, and turned the 

`Search for previous notes about this page when you open the extension menu?` setting ON.

Thanks! Not sure if I have misunderstood anything or missed a step in installation?

(also, I left the same issue on the Chrome extensions page, before I remembered to check Github.)

**Comment by fritzp6** [comment]
I have exactly the same problem, given exactly the same scenario

**Comment by fritzp6** [comment]
I got brave and messed with the default configuration by changing the "Append to existing note" template. I changed the POST setting from "API URL" to /vault/webclips/{{filename page.title}}.md. I clipped that setting from the "Create New Note" template.
After that change, I was able to append properly.

**Comment by viking2917** [comment]
@fritzp6 Awesome! That worked for me as well! Thank you! Not sure why the install/default setting does not work, but this at least solves the issue. (Note that if you have moved your web clippings to a new default directory (e.g. not webclips), one has to change /webclips to /whatever-your-directory-is.   (I reproduced the problem before I'd changed the default directory for new notes, but in my day-to-day use of this tool and Obsidian, I put them in a subdirectory with a different name).

Thanks again!

**Comment by coddingtonbear** [comment, marked as answer]
The "Append to existing note" is not quite what you're imagining it is -- it's a template that you can use with the "Note Recall" feature for appending to an existing note.  You do still have to use the "Note Recall" feature, though, to take advantage of it.

![image](https://user-images.githubusercontent.com/527661/220447882-d922666c-c197-4bfb-be16-5621d72823fb.png)

When using this feature and interacting with the icon in your toolbar, if an existing note is found, you can click the arrow icon shown for configuring obsidian web to use that template for adding new content:

![image](https://user-images.githubusercontent.com/527661/220448209-9169cd23-d1be-4e60-9fe1-70e147f9d260.png)

**Comment by fritzp6** [comment]
OK. I'll work on changing what I think should happen.
I restored the default settings and I got something to happen like I think
you describe, so I must be making progress.
Thank you for your help.
Fritz
